i18n/locales/en.ts

```diff
@@ -8,7 +8,7 @@ const translations = {
     coming_soon: 'Coming soon',
     start_chapter: 'Start chapter',
   },
-///ABOUT PAGE
+  ///ABOUT PAGE
```

Comment:
Not sure why we need this comment but we can remove it later

Reply:
These were added in https://github.com/saving-satoshi/saving-satoshi/pull/302

Reply:
They're just headings so contributors can more easily navigate the doc.